.github/workflows/readme-sync.yml

```diff
@@ -0,0 +1,53 @@
+name: Docs
+
+on:
+  pull_request:
+    paths:
+      - "docs/pydoc/**"
+  push:
+    branches:
+      - main
+      # release branches have the form v1.9.x
+      - "v[0-9].*[0-9].x"
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repo
+        uses: actions/checkout@v3
+
+      - name: Set up Python 3.8.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r docs/pydoc/requirements.txt
+
+      - name: Generate API docs
+        env:
+          # This is necessary to fetch the documentation categories
+          # from Readme.io as we need them to associate the slug
+          # in config files with their id.
+          README_API_KEY: ${{ secrets.README_API_KEY }}
+        run: ./.github/utils/pydoc-markdown.sh
+
+      - name: Get current version
+        id: current-version
+        if: github.event_name == 'push'
+        shell: bash
+        run: echo "__version__=$(grep -oP '__version__ = \"\K[0-9]+\.[0-9]+\.[0-9]+' __about__.py)" >> "$GITHUB_ENV"
```

Comment:
Wouldn't it be better to `print(__version__)` somehow?

Reply:
removed this since we use a fixed version